src/github/client/github-client.test.ts

```diff
@@ -15,7 +15,7 @@ class TestGitHubClient extends GitHubClient {
 	}
 }
 
-describe("GitHub Client", () => {
+describe("GitHub Client (mocks)", () => {
```

Comment:
Why?

Reply:
Cause in this file we mock the whole axios lib, however I've added one more to use it with nock